src/components/AuthPage/ForgotPassword/index.js

```diff
@@ -1,33 +1,27 @@
 import React, { useEffect, useState } from "react";
-import {
-  Alert,
-  Button,
-  Card,
-  Col,
-  Divider,
-  Form,
-  Input,
-  Row,
-  Typography,
-} from "antd";
-import { MailOutlined } from "@ant-design/icons";
+import MailOutlineOutlinedIcon from '@material-ui/icons/MailOutlineOutlined';
 import { Link } from "react-router-dom";
 import { useFirebase } from "react-redux-firebase";
 import { useDispatch, useSelector } from "react-redux";
 import { clearAuthError, sendPasswordResetEmail } from "../../../store/actions";
-const { Title } = Typography;
+import { Typography,Collapse,Button,Card,Grid,OutlinedInput,InputAdornment} from '@material-ui/core';
```

Comment:
Hi @rijusougata13,

Let's change the top level imports to path imports in Material-UI/Core components.

Convert,

```jsx
import { Button, TextField } from '@material-ui/core';
```
to

```jsx
import Button from '@material-ui/core/Button';
import TextField from '@material-ui/core/TextField';
```

This is done to decrease development startup time and to reduce the final React build bundle size. 

Read more about this 👇:
https://material-ui.com/guides/minimizing-bundle-size/

Reply:
changed top level imports to path imports.

Reply:
There is a new issue I found.



You have done this, (on `src/components/AuthPage/ForgotPassword/index.js` file line 14)

```jsx
import makeStyles from '@material-ui/core/styles/makeStyles';
```


According to the docs, they only support first and second level imports. So can you change it to,

```jsx
import { makeStyles } from "@material-ui/core/styles";
```

Refer documentation 👇:

![image](https://user-images.githubusercontent.com/25348766/119490448-7b772e00-bd7a-11eb-8a83-86fa7bdf51fe.png)
(from https://material-ui.com/guides/minimizing-bundle-size/)

Reply:
thanks for pointing out! fixed that problem. @Niweera

Reply:
You are welcome @rijusougata13. 👍